instances/atlas/brainAtlas/MarsAtlas.jsonld

```diff
@@ -0,0 +1,190 @@
+                  },
+                  {
+                        "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/PFrvl"
+                  },
+                  {
+                        "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/Pfrdli"
+                  },
+                  {
+                        "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/Pfrdls"
+                  },
+                  {
+                        "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/PFrd"
+                  },
+                  {
+                        "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/PFrm"
+                  },
+                  {
+                        "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/OFCvl"
+                  },
+                  {
+                        "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/OFCv"
+                  },
+                  {
+                        "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/OFCvm"
+                  },
+                  {
+                        "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/PFCvm"
+                  },
+                  {
+                        "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/Insula"
+                  },
+                  {
+                        "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/Thalamus"
+                  },
+                  {
+                        "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/Caudate"
+                  },
+                  {
+                        "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/Puttamen"
+                  },
+                  {
+                        "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/Pallidum"
+                  },
+                  {
+                        "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/Hippocampus"
+                  },
+                  {
+                        "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/Amygdala"
+                  },
+                  {
+                        "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/Accumbens"
+                  }
+            ]
+      },
+      "hasVersion": [
+            {
+                  "@id": "https://openminds.ebrains.eu/instances/brainAtlasVersion/Mars_v1_cortex"
+            },
+            {
+                  "@id": "https://openminds.ebrains.eu/instances/brainAtlasVersion/Mars_v2_cortexAndSubcortex"
+            }
+      ],
+      "homepage": "https://meca-brain.org/software/marsatlas/",
+      "shortName": "MarsAtlas",
+      "@context": {
+            "@vocab": "https://openminds.ebrains.eu/vocab/"
+      }
+}
```

Comment:
JSONs need a newline at the end to be standard conform (unfortunately Python does not do this automatically; you have to explicitly add a line when saving a JSON)

Reply:
fixed